debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/transforms/timescaledb/QueryInformationSchemaMetadata.java

```diff
@@ -52,7 +52,7 @@ public QueryInformationSchemaMetadata(Configuration config) {
         connection = new PostgresConnection(
                 JdbcConfiguration.adapt(config.subset(CommonConnectorConfig.DATABASE_CONFIG_PREFIX, true)
                         .merge(config.subset(CommonConnectorConfig.DRIVER_CONFIG_PREFIX, true))),
-                "Debezium TimescaleDB metadata");
+                "Debezium TimescaleDB metadata", "", "", "");
```

Comment:
TODO: This is okay for changes in our fork but would require passing the actual values in the upstream.